src/platform/startup_notify.rs

```diff
@@ -33,6 +34,28 @@ const X11_VAR: &str = "DESKTOP_STARTUP_ID";
 /// The variable which is used mostly on Wayland.
 const WAYLAND_VAR: &str = "XDG_ACTIVATION_TOKEN";
 
+/// The error type for when the requested operation is not supported by the backend.
+#[derive(Debug)]
+pub struct ActivationTokenNotFound {
+    _marker: (),
+}
+
+impl fmt::Display for ActivationTokenNotFound {
```

Comment:
Why it was changed to something that has nothing to do with the activation token. It can't be not found. It's either supported or not supported, which is why it was `NotSupportedError`.